spec/controllers/lambda_callback/resolution_proof_result_controller_spec.rb

```diff
@@ -11,22 +11,31 @@
 
       it 'accepts and stores successful resolution proofing results' do
         applicant = { first_name: Faker::Name.first_name, ssn: Faker::IDNumber.valid,
-                      zipcode: Faker::Address.zip_code }
+                      zipcode: Faker::Address.zip_code, state_id_number: '123',
+                      state_id_type: 'drivers_license', state_id_jurisdiction: 'WI' }
         document_capture_session.store_proofing_pii_from_doc(applicant)
-        proofer_result = ResolutionMock.new.proof(applicant)
+        Idv::Agent.new(applicant).proof_resolution(
+          document_capture_session,
+          should_proof_state_id: true,
+        )
+        proofer_result = document_capture_session.load_proofing_result[:result]
 
         post :create, params: { result_id: document_capture_session.result_id,
                                 resolution_result: proofer_result.to_h }
 
         proofing_result = document_capture_session.load_proofing_result
-        expect(proofing_result.result).to eq({ exception: '', success: 'true' })
+        expect(proofing_result.result).to include({ exception: '', success: 'true' })
```

Comment:
nit: I'd drop the optional curly braces
```suggestion
        expect(proofing_result.result).to include(exception: '', success: 'true')
```